src/main/mondrian/rolap/aggmatcher/ExplicitRecognizer.java

```diff
@@ -440,6 +447,17 @@ public int compare(
                 }
             });
     }
+
+    protected String getFactCountColumnName
+            (final JdbcSchema.Table.Column.Usage aggUsage) {
+        String measureName = aggUsage.getColumn().getName();
+        Map<String, String> measuresFactCount = tableDef.getMeasuresFactCount();
+        String factCountColumnName = measuresFactCount.get(measureName);
+        if (StringUtils.isEmpty(factCountColumnName)) {
```

Comment:
mondrian.olap.Util.isEmpty() is used in most places, probably best to stay consistent.